### DISCUSSION TITLE
如何禁用 service worker 缓存

### DISCUSSION THREAD

**Discussion by yooyea** [vuepress-theme-hope/vuepress-theme-hope#3636] [Q&A] [answered]
每次部署完都需要强刷才能获取最新内容

**Comment by Mister-Hope** [comment, marked as answer]
PWA 目前在模板中已经是默认关闭的状态，各种更新逻辑详见它的插件文档 https://plugin-pwa2.vuejs.press/zh/guide.html#%E6%9B%B4%E6%96%B0%E6%8E%A7%E5%88%B6。

如果之前部署了pwa需要清除，使用 remove-pwa 插件 https://plugin-remove-pwa.vuejs.press/zh/。
